molgenis/molgenis-emx2
# feat: beacon customize informational endpoint data

What are the main changes you did:
Added a endpoint data table to the data profiles, and added it to the beacon v2 profile.
Fetch the data for a beacon info request :schema/api/beacon/info from this table.
Addresses issue #3661 

how to test:
- Go to preview server: https://preview-emx2-pr-3870.dev.molgenis.org/
- Add a schema using the beacon v2 demo data
- Customize the data in the Endpoint table
- go to [schema_name]/api/beacon/info
- See the custom info

todo:
- [ ] Handle response when no endpoint info is present (default settings?)
- [ ] Update other informational endpoints similarly
